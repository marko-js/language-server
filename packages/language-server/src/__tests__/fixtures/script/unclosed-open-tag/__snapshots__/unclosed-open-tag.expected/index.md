## Diagnostics
### Ln 1, Col 6
```marko
> 1 | <div hi
    |      ^ EOF reached while parsing attribute "hi" for the "div" tag
  2 |
```

### Ln 1, Col 6
```marko
> 1 | <div hi
    |      ^^ Object literal may only specify known properties, and '"hi"' does not exist in type 'Directives & Div'.
  2 |
```

