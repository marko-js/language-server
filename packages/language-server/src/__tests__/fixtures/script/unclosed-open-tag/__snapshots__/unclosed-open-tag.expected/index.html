<div data-marko-node-id="0" hi></div>